frontend/src/components/ProjectServiceAccountRow.vue

```diff
@@ -51,6 +52,13 @@ limitations under the License.
           {{username}}
         </v-list-tile-sub-title>
       </v-list-tile-content>
+      <v-list-tile-action>
+        <v-layout row align-center>
+          <v-chip v-for="(roleName, index) in roleNames" :key="index" small color="blue-grey darken-2" text-color="white">
```

Comment:
in the backend we talk about role and roles. What is the difference between `role` and `roleName`? Use `roleName` as key.

Reply:
refactored to roleDisplayName